src/Symfony/Component/VarDumper/Caster/MemcachedCaster.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Caster;
+
+use Symfony\Component\VarDumper\Cloner\Stub;
+
+/**
+ * @author Jan Schädlich <jan.schaedlich@sensiolabs.de>
+ */
+class MemcachedCaster
+{
+    public static function castMemcached(\Memcached $c, array $a, Stub $stub, $isNested)
+    {
+        $servers = array();
+        foreach ($c->getServerList() as $key => $serverData) {
+            $servers['server_'.$key] = new EnumStub($serverData);
+        }
+
+        $a += array(
+            Caster::PREFIX_VIRTUAL.'servers' => new EnumStub(
```

Comment:
Maybe this one isn't needed?

Reply:
(the EnumStub I mean)

Reply:
I thought the dump output will look nicer with the `EnumStub` around it:

```
Memcached {
  servers: {
    server_0: {
      host: "127.0.0.1"
      port: 11211
      type: "TCP"
    }
    server_1: {
      host: "127.0.0.2"
      port: 11211
      type: "TCP"
    }
  }
...
}
```
vs.

```

Memcached {
  servers: array:2 [
    "server_0" => {
      host: "127.0.0.1"
      port: 11211
      type: "TCP"
    }
    "server_1" => {
      host: "127.0.0.2"
      port: 11211
      type: "TCP"
    }
  ]
...
}
```

Reply:
Ok, I will remove the surrouding `EnumStub`.